packages/mermaid/src/diagrams/info/infoDiagram.ts

```diff
@@ -1,13 +1,12 @@
-import { DiagramDefinition } from '../../diagram-api/types.js';
-// @ts-ignore: TODO Fix ts errors
+import type { DiagramDefinition } from '../../diagram-api/types.js';
+// @ts-ignore jison doesn't export types
 import parser from './parser/info.jison';
 import db from './infoDb.js';
-import styles from './styles.js';
 import renderer from './infoRenderer.js';
 
 export const diagram: DiagramDefinition = {
-  parser,
-  db,
-  renderer,
-  styles,
+  parser: parser,
+  db: db,
+  renderer: renderer,
+  styles: () => '',
```

Comment:
Well do.

Do you have anything on your mind about strict styling definitions? I’m sure that the current approach of writing CSS code as strings with no type safety or highlighting or etc.

I came across [csstypes](https://github.com/frenic/csstype), not sure about bundling size or limitations. Do you have something in mind? I’m almost done with pie chart files, but I'm not sure what to do with the `styles.js` file yet.

Reply:
And since I'm already changing the pie renderer, I might resolve this issue #3987 with it since it's just a mathematical error, not a svg side, but I'm not sure what is the best approach yet. Adding decimals to the chart would lead to more problems and break almost all existing Cypress tests.

Do you something in your mind? Or should I just leave and focus on converting files and adding types without resolving existing error?

Reply:
> Do you something in your mind? Or should I just leave and focus on converting files and adding types without resolving existing error?

Let's keep this PR narrowly scoped to just fixing types. It might be the best to leave https://github.com/mermaid-js/mermaid/issues/3987 alone. Or we'd have to add a config for decimals.

> I came across [csstypes](https://github.com/frenic/csstype) and [vanilla-extract](https://github.com/vanilla-extract-css/vanilla-extract), not sure about bundling size or limitations. Do you have something in mind? I’m almost done with pie chart files, but I'm not sure what to do with the styles.js file yet.

I think [vanilla-extract](https://github.com/vanilla-extract-css/vanilla-extract) looks great, and we can use that as it won't be bundled because it's a build time dependency. 

But this also should be a PR of it's own as we could discuss in depth on the implications there.

Reply:
> Let's keep this PR narrowly scoped to just fixing types. It might be the best to leave #3987 alone. Or we'd have to add a config for decimals.

Okey, I'll be focusing more on converting files and adding types with JSDoc comments.

> I think [vanilla-extract](https://github.com/vanilla-extract-css/vanilla-extract) looks great, and we can use that as it won't be bundled because it's a build time dependency.
> 
> But this also should be a PR of it's own as we could discuss in depth on the implications there.

Sound great!